components/Tabs/README.md

```diff
@@ -297,10 +297,7 @@ id<MDCTypographyScheming> typographyScheme = [[MDCTypographyScheme alloc] init];
 
 ### Theming Extensions (Beta)
```

Comment:
Delete Beta here and throughout.

Reply:
Addressed in 81204fc0f4cfc6e0b0d8b2b2c08f4532a91e4672.